app/assets/stylesheets/partials/_typography.scss

```diff
@@ -36,6 +36,11 @@ p.font-xsmall{
   display: block;
 }
 
+.govuk-list, .app-prose-scope ul {
+    padding-left: 20px;
+    list-style-type: disc;
```

Comment:
This is used across multiple areas on the service, can you check this has no impact on other screens?